css/cssom/getComputedStyle-pseudo-with-argument.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>CSSOM: Handling pseudo-elements with arguments</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<link rel=help href=https://drafts.csswg.org/cssom/#dom-window-getcomputedstyle>
+<style>
+#pseudo-invalid::highlight(name) {
+  color: rgb(0, 128, 0);
+}
+#pseudo-invalid::view-transition-image-pair(name) {
+  color: rgb(0, 128, 0);
+}
+#pseudo-invalid::view-transition-group(name) {
+  color: rgb(0, 128, 0);
+}
+#pseudo-invalid::view-transition-old(name) {
+  color: rgb(0, 128, 0);
+}
+#pseudo-invalid::view-transition-new(name) {
+  color: rgb(0, 128, 0);
+}
+</style>
+<ul><li id="pseudo-invalid">Item</li></ul>
+<script>
+[
+  "::before(test)",
+  "::highlight",
+  "::highlight(",
+  "::highlight()",
+  "::highlight(1)",
+  "::highlight($)",
+  "::highlight (name)",
+  "::highlight(name)a",
+  "::view-transition-group(*)",
+].forEach(nonParsablePseudoIdentifier => {
+  test(() => {
+    const li = document.querySelector('li');
+    assert_equals(getComputedStyle(li, nonParsablePseudoIdentifier).length, 0);
+  }, `This pseudo-element should not parse: ${nonParsablePseudoIdentifier}`)
+});
+
+[
+  "::highlight(name)",
+  "::highlight(\nname",
```

Comment:
How come this should parse without an ending )?

Reply:
Because that's how the CSS parser works. `<style>body { background:lime</style>` also works.

Reply:
Right, forgot about this... but perhaps having parsable cases that also have a `)` at the end wouldn't hurt?

Reply:
That's the first one, no?

Reply:
No, I meant both whitespace and an ending )